syntax = "proto2";

package com.google.tuningfork;

enum Level {
  // Enums must start at 1
  LEVEL_1 = 1;
  LEVEL_2 = 2;
  LEVEL_3 = 3;
};

message Annotation {
  optional Level level = 1;
  optional Level next_level = 2;
}

enum LOD {
  // Enums must start at 1
  LOD_1 = 1;
  LOD_2 = 2;
  LOD_3 = 3;
  LOD_4 = 4;
};

message FidelityParams {
  optional LOD lod = 1;
}
